cloudslang-cli/src/test/java/io/cloudslang/lang/cli/utils/CompilerHelperTest.java

```diff
@@ -101,6 +104,48 @@ public void testDependenciesFileParentFolder() throws Exception {
                         SlangSource.fromFile(flowPath)
                 )
         );
+        Mockito.verify(slang).compileCleanUp();
+    }
+
+    @Test
+    public void testCompileSourceCleanup() throws Exception {
+        URI flowPath = getClass().getResource("/executables/dir3/flow.sl").toURI();
+        URI opPath = getClass().getResource("/executables/dir3/dir3_1/test_op.sl").toURI();
+        compilerHelper.compileSource(flowPath.getPath(), null);
+        Mockito.verify(slang).compileSource(
+                SlangSource.fromFile(flowPath),
+                Sets.newHashSet(
+                        SlangSource.fromFile(opPath),
+                        SlangSource.fromFile(flowPath)
+                )
+        );
+        Mockito.verify(slang).compileCleanUp();
+    }
+
+    @Test
+    public void testCompileFoldersCleanup() throws Exception {
+        URI folderPath = getClass().getResource("/executables/dir3").toURI();
+        List<String> folders = new ArrayList<>();
+        folders.add(folderPath.getPath());
+        compilerHelper.compileFolders(folders);
+
+        URI flowPath = getClass().getResource("/executables/dir3/flow.sl").toURI();
+        URI opPath = getClass().getResource("/executables/dir3/dir3_1/test_op.sl").toURI();
+        Mockito.verify(slang).compileSource(
```

Comment:
static import for `Mockito.verify` and others from Mockito will make the test code less verbose